sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeActionsAsyncClient.java

```diff
@@ -245,6 +248,8 @@ private JobManifestTasks getJobManifestTasks(TextAnalyticsActions actions) {
                         entityLinkingTask.setParameters(
                             new EntityLinkingTaskParameters()
                                 .setModelVersion(action.getModelVersion())
+                                .setLoggingOptOut(action.isServiceLogsDisabled())
+                                .setStringIndexType(StringIndexType.UTF16CODE_UNIT)
```

Comment:
keep consistency with other action behavior.

Reply:
consider adding the addition of string index type default to utf16 for entity linking in the changelog

Reply:
same for sentiment